pkg/tfvars/gcp/gcp.go

```diff
@@ -15,8 +15,9 @@ const (
 
 // Auth is the collection of credentials that will be used by terrform.
 type Auth struct {
-	ProjectID      string `json:"gcp_project_id,omitempty"`
-	ServiceAccount string `json:"gcp_service_account,omitempty"`
+	ProjectID        string `json:"gcp_project_id,omitempty"`
+	NetworkProjectID string `json:"gcp_network_project_id,omitempty"`
```

Comment:
This field doesn't really belong in the `Auth` structure.... I recommend placing it into the `config` structure below.

Reply:
I'm actually on the fence; I don't believe it's worth blocking on this.